lib/posix/stropts.h

```diff
@@ -0,0 +1,135 @@
+#ifndef _STROPTS_H
```

Comment:
this file should be in `zephyr/include/zephyr/posix/`?

Reply:
Realised that now! Fixing, thanks!

Reply:
It should have a copyright at the top. Apache 2.0 spdx

Reply:
Gotcha, adding that in. Quick question: Who should I attribute the copyright to? Since the standard was specified by opengroup.

Reply:
You